src/test/java/giis/eshopcontainers/e2e/functional/model/Order.java

```diff
@@ -1,9 +1,9 @@
 package giis.eshopcontainers.e2e.functional.model;
-import java.util.List;
-
 
+import java.util.List;
 
 public class Order {
```

Comment:
The review didn't include restructuring the models. Get them back to their state before the review.

NOTE: Never add unrelated changes after a revision because this makes extremely difficult the review process. It is good to improve the code, but this must be handled in a separate PR.

Reply:
I don't see the changes, the only change since last review is this:
![image](https://github.com/giis-uniovi/retorch-st-eShopContainers/assets/10879637/c197a067-bfbb-49e5-af42-dc02e656355a)

Reply:
For some reason my IDE reverted to the wrong commit (the previous one instead the original). Sorry for the inconveniences, now the model files are in the original state (the first commit)